src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
 // write code here
+const employeeList = document.querySelectorAll('li');
+
+function sortList(list) {
+  const employees = getEmployees(list);
+
+  employees.sort(
+    (a, b) => convertSalaryToNumber(b.salary) - convertSalaryToNumber(a.salary)
+  );
+
+  for (let i = 0; i < employees.length; i++) {
+    list[i].dataset.position = employees[i].position;
+    list[i].dataset.salary = employees[i].salary;
+    list[i].dataset.age = employees[i].age;
+    list[i].textContent = employees[i].name;
```

Comment:
Do not mutate current dom tree, apent new children

Reply:
Ok, done